homeassistant/components/sleepiq/entity.py

```diff
@@ -26,20 +40,14 @@ def __init__(
         sleeper: SleepIQSleeper,
         name: str,
     ) -> None:
-        """Initialize the SleepIQ side entity."""
+        """Initialize the SleepIQ sensor entity."""
         super().__init__(coordinator)
-        self.bed = bed
+        SleepIQEntity.__init__(self, bed)
```

Comment:
Please separate this into multiple base classes to avoid multiple `__init__` calls